src/main/java/io/passport/server/controller/SurveyController.java

```diff
@@ -31,6 +31,18 @@ public SurveyController(SurveyService surveyService) {
         this.surveyService = surveyService;
     }
 
+    /**
+     * Read all surveys
+     * @return
+     */
+    @GetMapping("/all")
```

Comment:
Let's remove "/all" from here. Make it empty string so that it will be aligned with other endpoints.